docs/01-apps/01-game-client/01-product/01-product-overview.md

```
@@ -0,0 +1,35 @@
+---
+description: Product overview description
+---
+
+## Overview
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)